docs/DFAnumerics.rst

```diff
@@ -607,3 +610,69 @@ Finaly, the entrainment force reads:
 
 .. math::
     F_{k,i}^{\text{ent}} = -w_f\,(e_s+\,q_{k}^{\text{ent}}\,e_d)
+
+Adding forces
+--------------
+The different components are added following an operator splitting method.
+This means, the velocity of the particles are updated successively with the different forces.
+
+
+Adding artificial viscosity
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+If the viscosity option (``viscOption``) is set to 1, the artificial viscosity term is added first as described
+in :ref:`DFAnumerics:Artificial viscosity`. With ``viscOption`` set to 0, no viscosity is added. Finally, if
+``viscOption`` is set to 2, the artificial viscosity is added together with computing the SPH force (add link to description)
+
+
+Adding entrainment
+~~~~~~~~~~~~~~~~~~~
+Then, the entrainment is taken into account by adding first the component representing the loss of momentum due to
+the need to accelerate the entrained mass (:math:`- \overline{u}_{k,i}\,A^{\text{ent}}_{k}\,q^{\text{ent}}_{k}`) followed by adding the force due to the need to break and compact the
+entrained mass (:math:`F_{k,i}^{\text{ent}}`) as described in :ref:`DFAnumerics:Entrainment force`.
```

Comment:
```suggestion
Entrainment is taken into account by first adding the component representing the loss of momentum due to
acceleration of  the entrained mass :math:`- \overline{u}_{k,i}\,A^{\text{ent}}_{k}\,q^{\text{ent}}_{k}`.  Second by adding the force due to the need to break and compact the
entrained mass (:math:`F_{k,i}^{\text{ent}}`) as described in :ref:`DFAnumerics:Entrainment force`.
```
Not sure what you mean by 'need to break'

Reply:
erode?

Reply:
when it is plowing I dont know if we can talk of errosion